### PR TITLE
Minecraft modal

### PR DESCRIPTION
**Pull request by Zerrien**
Minecraft modal

### CODE REVIEW

File: app/templates/admin/analytics.jade [hunk 1]
```diff
@@ -36,7 +36,7 @@ block content
                 - homeBigMAU += count;
             div.count= homeBigMAU
 
-    ul.nav.nav-tabs
+    ul.nav.nav-tabs(style="border:none")
```
Comment (nwinter) [inline app/templates/admin/analytics.jade]: Eww, style in the template!

File: app/templates/core/mine-modal.jade [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        #capacity-text
+          div We are not currently accepting additional beta users for CodeCombat Minecraft, but hope to make more available in the future.
+          form#notify-me-form
+            .checkbox
+              label
+                // NOTE: This needs to be written as HTML or it won't pick up contained text
+                <input id="notify-me-check" type="checkbox"> Yes, please notify me about the CodeCombat Minecraft beta
+        #button-container
+          button#buy-now-button.btn.btn-lg.btn-illustrated.purchase-button
+            span Buy Now $9.99/
```
Comment (nwinter) [inline app/templates/core/mine-modal.jade]: Nicer to load this from server

File: app/templates/admin/analytics.jade (hunk 1 above)
Comment (Zerrien) [inline app/templates/admin/analytics.jade]: As suggested by Matt, accidentally got wrapped into a commit. (Fixes the stray line above our analytics graph on the admin/analytics page 😄 )

**Comment by nwinter** [comment]
Hmm, why did coverage drop so much there? Build or tests failing for Coveralls but not Travis?

**Pull request merged**